d2l-icon-button.html

```diff
@@ -18,7 +18,8 @@
 			behaviors: [window.D2L.PolymerBehaviors.IconButtonBehavior],
```

Comment:
My personal preference is for using `title`, since it is natural for putting that as the tool tip text, and the coding is simpler.

I'm fine with either way, but yeah, I agree we need to pick an approach and stick with it.